app/src/main/java/com/nextcloud/talk/conversationlist/ConversationsListActivity.kt

```diff
@@ -869,31 +877,39 @@ class ConversationsListActivity :
         binding?.newMentionPopupBubble?.let { viewThemeUtils.material.colorMaterialButtonPrimaryFilled(it) }
     }
 
+
     @Suppress("Detekt.TooGenericExceptionCaught")
     private fun checkToShowUnreadBubble() {
-        try {
-            val lastVisibleItem = layoutManager!!.findLastCompletelyVisibleItemPosition()
-            for (flexItem in conversationItems) {
-                val conversation: Conversation = (flexItem as ConversationItem).model
-                val position = adapter!!.getGlobalPositionOf(flexItem)
-                if (hasUnreadItems(conversation) && position > lastVisibleItem) {
-                    nextUnreadConversationScrollPosition = position
-                    if (!binding?.newMentionPopupBubble?.isShown!!) {
-                        binding?.newMentionPopupBubble?.show()
+        searchBehaviorSubject.subscribe { value ->
```

Comment:
lint complains about "The result of subscribe is not used". 
I guess for this case it's okay to add
@SuppressLint("CheckResult")